js/ex-valInt.mjs

```diff
@@ -0,0 +1,36 @@
+import * as R from "ramda";
+
+/* Creare una funzione (n, lower, upper) 
+che valida se un numero n è all interno di 
+un range definito da lower e upper .
+
+intInArray(3, 1, 9) ➞ true
+
+intInArray(6, 1, 6) ➞ false */
+
+console.log('----JavaScript-----')
+
+const validateInt = (n, lower, upper) => { 
+    if (n>lower && n<upper && R.is(Number, n)) {
```

Comment:
le condizioni vengono eseguite in ordine quindi conviene posizionare come primo quello sul controllo del numero